magic-modules/generate_terraform.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Setup GOPATH
+export GOPATH="${PWD}/go"
+WORKDIR="${PWD}"
+
+set -x
+set -e
+
+# Create GOPATH structure
+mkdir -p "${GOPATH}/src/github.com/terraform-providers"
+
+cd magic-modules
```

Comment:
Prefer not to - since I don't get to know what directory I start in, it'll just be changing a bunch of `cd foo` to `cd ${WORKDIR}/foo`, which doesn't add additional information.  Given that I don't know what directory I start in, still think it's worth it?

Reply:
Chose to use `pushd'/'popd`, I prefer that.